UICatalog/Scenarios/TabViewExample.cs

```diff
@@ -68,6 +68,7 @@ public override void Setup ()
 				 new Label ("This tab has a very long name which should be truncated.  See TabView.MaxTabTextWidth")),
 				 false);
 			tabView.AddTab (new Tab ("Les Mise" + Char.ConvertFromUtf32 (Int32.Parse ("0301", NumberStyles.HexNumber)) + "rables", new Label ("This tab name is unicode")), false);
+			tabView.AddTab (new Tab ("Les Mise" + Char.ConvertFromUtf32 (Int32.Parse ("0328", NumberStyles.HexNumber)) + Char.ConvertFromUtf32 (Int32.Parse ("0301", NumberStyles.HexNumber)) + "rables", new Label ("This tab name has two unicode combining masks")), false);
```

Comment:
Simpler:

```cs
tabView.AddTab (new Tab ("Les Mise" + '\u0301' + "rables", new Label ("This tab name is unicode")), false);
tabView.AddTab (new Tab ("Les Mise" + '\u0328' + '\u0301' + "rables", new Label ("This tab name has two unicode combining masks")), false);
```

Also, I think it woudl be better to expand the Unicode scenario than to bury these tests in TabView Example, no?

Reply:
The glyph that's supposed to appear with `e\u0328\u0301` is 
![image](https://github.com/gui-cs/Terminal.Gui/assets/585482/fbe4b125-ab96-4a7b-8cdb-d545eddf2cc8)

Reply:
Well I think I already told you that the string builder appear with the expected unicode but not in the screen. You said that terminal doesn't support it. Do you already saw this https://github.com/gui-cs/Terminal.Gui/pull/2932#discussion_r1372303371?
It's very explicitly explained that the output code is right but the `Console.Write` doesn't render it well.